Updates for arm64 builds.

glevand opened the pull request.

mischief commented on profiles/coreos/arm64/package.accept_keywords:
```diff
@@ -53,6 +53,8 @@ net-fs/nfs-utils ~arm64
 =sys-apps/smartmontools-6.4 **
 
+=app-misc/jq-1.4-r1 **
+=net-misc/curl-7.45.0 ~arm64
```
are these not sufficient?

https://github.com/coreos/coreos-overlay/blob/master/profiles/coreos/base/package.accept_keywords#L59
https://github.com/coreos/coreos-overlay/blob/master/profiles/coreos/base/package.accept_keywords#L95

glevand commented:
@mischief  I had been using a partially updated source base.  The upstream profile works OK for curl and jq so I removed those. I did need to add unstable bison for arm64.  I also updated the arm64 kernel config to enable CONFIG_IFB=y.

mischief commented:
lgtm

glevand commented:
Rebased to coreos-kernel-4.4.4.  Will commit after final test.

mischief commented:
thanks!
